packages/driver/src/cypress/commands.coffee

```diff
@@ -115,6 +116,12 @@ create = (Cypress, cy, state, config) ->
         obj = options
         options = {}
 
+      ## this is a bit of a hack. we need to know if a command is built-in or
+      ## custom for error display purposes. almost all built-in commands go through
+      ## this method and it's undocumented for users, so we use it to mark them
+      ## as not being custom
+      options.isCustom = false
```

Comment:
maybe `isBuiltIn` or something, since its the default, typically associated with `true` as opposed to "off", which is the lesser likely scenario